Update community sample (#129)
IBM/charts: CONTRIBUTING.md

This repository hosts helm charts intended for use with IBM(R) Cloud Private.

> **Important:** all contributions from outside of IBM must be made to the `community` directory.  IBM teams may also contribute Helm charts for open source projects in the `community` directory.  However, the `stable` directory is used only for IBM product charts distributed by IBM.  IBM product teams must continue to deliver charts for IBM products through the existing internal build process, and cannot add charts directly through GitHub.

Reviewer: Minor style nag - typically, "however" shouldn't be used to start sentences.
```suggestion
**Important:** the `stable` directory cannot be directly contributed to through GitHub and is used only for IBM product charts that go through IBM's internal build process. If you are contributing charts for non-IBM products, or are from an IBM team contributing charts for open source projects, these contributions must be made to the `community` directory.
```
